build/references/command-line-interface.md

```diff
@@ -341,27 +349,62 @@ Portion of pending message buffer reserved for messages from validators. Default
 
 Portion of chain’s CPU time reserved for messages from validators. Defaults to `0.375`.
 
-### Network Timeout
+### Network
 
 `--network-initial-timeout` \(duration\):
 
 Initial timeout value of the adaptive timeout manager, in nanoseconds. Defaults to `5s`.
 
 `--network-minimum-timeout` \(duration\):
 
-Minimum timeout value of the adaptive timeout manager, in nanoseconds. Defaults to `5s`.
+Minimum timeout value of the adaptive timeout manager, in nanoseconds. Defaults to `2s`.
 
 `--network-maximum-timeout` \(duration\):
 
 Maximum timeout value of the adaptive timeout manager, in nanoseconds. Defaults to `10s`.
 
-`--network-timeout-multiplier` \(float\):
+`--network-timeout-halflife` `\(duration\):
```

Comment:
Can you please remove the single backticks before duration and anywhere else there are improperly closed code blocks?